chore: align README for non-prefixed packages (#2994)
vaadin/web-components: packages/tabs/README.md

# @vaadin/tabs

> A web component used to organize and group content into sections.

Reviewer: ```suggestion
A web component for organizing and grouping content into sections.
```

Author: Done.
